lib/streamlit/elements/metric.py

```diff
@@ -29,8 +29,8 @@
     from streamlit.delta_generator import DeltaGenerator
 
 
-Value: TypeAlias = Union["np.integer", "np.floating", float, str, None]
-Delta: TypeAlias = Union[float, str, None]
+Value: TypeAlias = Union["np.integer", "np.floating", float, int, str, None]
+Delta: TypeAlias = Union[float, int, str, None]
```

Comment:
nit: are we not supporting `"np.integer", "np.floating",` for Delta as well?

Reply:
Hm good point. I'll add those

Reply:
Hmm, looking at this a bit more, we actually do extra stuff for numpy values here: https://github.com/streamlit/streamlit/blob/69d20270af9ea1fd44e3fd3f9e71a5970f1532a1/lib/streamlit/elements/metric.py#L143

Reply:
I have decided to make this a separate PR.